conf/app.routes

```diff
@@ -23,7 +23,8 @@ GET         /account/sign-out                               controllers.auth.Aut
 GET         /register/problem/signed-out                    controllers.auth.SignedOutController.onPageLoad()
 
 GET         /unauthorised                                   controllers.UnauthorisedController.onPageLoad
-
+GET        /register/problem/different-business             controllers.DifferentBusinessController.onPageLoad()
```

Comment:
are these url's supposed to have /problem/ in the title?
often the full prototype url isn't needed

Reply:
Yes as we are following same for other services as well like in mar and crs , so basically we are following a standard so hoplly keeping the url as per prototype would be a right approach.